### ISSUE TITLE
Profile the game loop

### ISSUE THREAD

**Issue by mlange-42** [mlange-42/tiny-world#236]
Not that there are any perfoemance issues... However, it would be interesting to check where the game spends the most time, i.e. to save a bit of energy.

**Comment by Ecostack** [comment]
How would you go about profiling? Would be `pprof` enough in that case?

**Comment by mlange-42** [comment]
Yes, that's what I was intending. For examples, see the Arche profiling stuff. Particularly the main files nested under https://github.com/mlange-42/arche/tree/main/benchmark/profile.

I usually use the graph visualization. Flame graph might also be helpful.
